relayer/__init__.py

```diff
@@ -0,0 +1,33 @@
+from kafka import KafkaProducer
+
+from .event_emitter import EventEmitter
+from .exceptions import NonJSONSerializableMessageError, UnsupportedPartitionKeyTypeError, ConfigurationError  # noqa
```

Comment:
We're using them but not on this file, those are being exported so they can easily being imported by the client.